.github/workflows/publish-hapi-api-protos.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  release:
+    types: [published]
+
+name: hapi-api-publish
+jobs:
+  npm-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci
+      - run: npm run build-proto
+      - run: npm version patch
+      - run: npm pack
+      - run: npm publish
```

Comment:
I think Travis said we should use github's npm registry for this, is that what this will do?

Reply:
Yeah, it should push to Github's npm. It's in the package.lock package configuration.